src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/Example2.txt

```diff
@@ -12,4 +12,6 @@ public void foo(String s) {} // OK
 public void notFoo() {} // violation. Have to be after foo(String s, int i)
 public void foo(int i, String s) {}
 public void foo(String s, int i) {}
+private interface Testing() {}
+public void foo(double d) {} // violation. Have to be after foo(String s, int i)
```

Comment:
This new stuff to the example isn't completely making sense.

I understand why you added `interface` to both, but there is no example of `foo(double d)` in Example 1. So you lose the reason why you added interface to both if your not keeping everything else in synch too.

This needs to be fixed up a bit more.

Reply:
Done. Here is the new changes: https://github.com/checkstyle/checkstyle/blob/24ab0e93ad43a656fca70f6a7fd16b9d559bafdf/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/Example2.txt#L10-L15